src/Cache.js

```diff
@@ -62,11 +62,13 @@ class Cache {
    * remove from cache after maxAge time has passed
    *
    * @param {*} key the key to remove
-   * @param {number} maxAge the time in milliseconds to wait before removing key
+   * @param {number} maxAge the time in milliseconds to wait before removing the key
+   * @param {Function} onExpiration a callback that is called after removing the key
    */
-  expireAfter(key: any, maxAge: number) {
+  expireAfter(key: any, maxAge: number, onExpiration: ?Function) {
     setTimeout(() => {
       this.remove(key);
+      if (onExpiration) onExpiration(key.key);
```

Comment:
so it looks like the prettier experiment is over ... this is one of the style rules that should have been caught in linting. all if blocks should be brace encased:

```javascript
if (onExpiration) {
  onExpiration(key.key);
}
```